src/mlpack/tests/policy_gradient_test.cpp

```diff
@@ -108,7 +108,7 @@ TEST_CASE("PendulumWithGaussianDDPG", "[PolicyGradientTest]")
     // Set up the GaussianNoise parameters.
     int size = 1; 
     double mu = 0.0; 
-    double sigma = 0.1; 
+    double sigma = 0.01;
```

Comment:
Just digging into the code here to make sure these changes are valid, I noticed that `GaussianNoise` internally uses `arma::randi` instead of `arma::randn`.  @tareknaser I see your name is on the file, do you know if that's a typo or is it intentional that we are sampling integers there?

https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/reinforcement_learning/noise/gaussian.hpp#L48

Reply:
Yes, it's a typo
`arma::randn` makes more sense

Reply:
Sounds good, thanks! :+1: